tsconfig.json

```diff
@@ -8,13 +8,12 @@
     "strict": true,
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
+    "resolveJsonModule": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "baseUrl": "./",
     "paths": {
-      "astarte-client": [
-        "./src/astarte-client/index.ts"
```

Comment:
Is there a reason for this change? It feels like recently we're adding a lot of fomatting changes in PRs that shouldn't bother with them.
Please check your format on save options or carefully add hunks to your commit with `git add -p`

Reply:
I was actually aware of the change, since I formatted the file along with the real modification. But your point is good one, I'll avoid doing this in the future